tests/integration/ledger_sync/ls_outboundq_tests.py

```diff
@@ -25,14 +25,36 @@
     delete_role_by_name,
     delete_user_by_username,
     get_outbound_queue_depth,
+    get_outbound_queue_entry,
     get_proposal_with_retry,
+    get_role,
     log_in,
     add_role_member,
 )
 
 LOGGER = get_default_logger(__name__)
 
 
+def prepare_outbound_queue_data(entry, data_type):
```

Comment:
if the next_id and role_id are both internally generated UUIDs, couldn't we just filter using `default` and `or` operators in REQL? https://rethinkdb.com/api/python/or/

Reply:
on second thought, using `default` seems to require using `lambda` inside of filter, which will cause a  pretty good performance hit. Does anyone know why we don't just call the NEXT generated UUID the `next_id` in all objects? we might want to switch to a more consistent naming scheme like that down the road, it would eliminate the need for this sort of duplication

Reply:
I agree that switching the naming scheme to be more consistent down the road would help us with removing this sort of duplication in logic. I think we just never got to changing role_ids to next_ids when we were changing to user_id to next_id.